src/DataFixtures/ORM/LoadCommitteeCandidacyData.php

```diff
@@ -77,18 +78,64 @@ public function load(ObjectManager $manager)
 
         $manager->persist($candidacy);
 
+        $adherent = $this->getReference('senatorial-candidate');
+        $committee = $this->getReference('committee-12');
+
+        // Designation COMMITTEE-SUPERVISOR
+        $this->createSupervisorCandidacy(
+            $manager,
+            $committee,
+            $adherent,
+            $this->getReference('municipal-manager-1'),
+            __DIR__.'/../../../app/data/dist/avatar_homme_01.jpg'
+        );
+
         $manager->flush();
     }
 
     public function getDependencies()
     {
         return [
             LoadAdherentData::class,
```

Comment:
`LoadAdherentData` is no longer required here, because `LoadCommitteeData` depends on `LoadAdherentData`